modules/finance/src/test/java/com/opengamma/strata/finance/rate/bond/FixedCouponBondTradeTest.java

```diff
@@ -74,6 +77,7 @@
       .build();
   private static final Security<FixedCouponBond> BOND_SECURITY =
       UnitSecurity.builder(PRODUCT).standardId(SECURITY_ID).build();
+  private static final Payment UPFRONT_PAYMENT = Payment.of(CurrencyAmount.of(EUR, -NOTIONAL * 0.065), SETTLEMENT_DATE);
```

Comment:
Tests should have realistic values: payment amount = -NOTIONAL * QUANTITY * 0.99 or something similar.